src/useIsStrictMode.ts

```diff
@@ -4,17 +4,23 @@ import * as React from "react";
  * @returns Current react fiber being rendered
  */
 export const getCurrentOwner = () => {
+  // Note: String concatenation is used to prevent bundlers to complain with multiple versions of React
   try {
     // React 19
-    // @ts-ignore - using react internals
-    return React.__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE.A.getOwner();
+    // using react internals
+    return (React as any)[
+      "__".concat(
+        "CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE"
+      )
```

Comment:
```suggestion
      "".concat(
        "__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE"
      )
```

then just `"".concat()` should also work? Please check, if it works, please update and we will be ready to merge :)

Reply:
@layershifter  Thank you very much. This is an easier to understand workaround, and it also works well with my webpack bundler!

I have committed this fix.